biolark/pom.xml

```diff
@@ -86,6 +86,7 @@
       <groupId>net.sf.json-lib</groupId>
       <artifactId>json-lib</artifactId>
       <classifier>jdk15</classifier>
+      <version>2.3</version>
```

Comment:
Hmm after looking at it, I don't think the code actually uses `json-lib` at all, so we could probably take off that dependency.